docs/resources/trigger.md

```diff
@@ -110,40 +110,40 @@ resource "wallarm_trigger" "attack_trigger" {
 
 ## Argument Reference
 
-* `client_id` - (Optional) ID of the client to apply the trigger to. The value is required for multi-tenant scenarios.
-* `template_id` - (Required) Trigger condition. A condition is a system event to be notified about. Can be:
+* `client_id` - (optional) ID of the client to apply the trigger to. The value is required for [multi-tenant scenarios][2].
+* `template_id` - (**required**) trigger condition. A condition is a system event to be notified about. Can be:
   - `user_created` for a user added to the company account in Wallarm Console
   - `attacks_exceeded` for detected attacks number exceeded the specified value
   - `hits_exceeded` for detected hits number exceeded the specified value
   - `incidents_exceeded` for detected incidents number exceeded the specified value
   - `vector_attack` for detected attack vectors number exceeded the specified value
   - `bruteforce_started` for detected attack to be identified as bruteforce
```

Comment:
it is the number... please check the real trigger behavior and update the sentence

Reply:
@aoleynikov-wallarm, could you please provide the example where `template_id` is set to `bruteforce_started` - will there be any numbers in the configuration? UI suggests "more than 30 requests from the same IP per 30s" by default.